core/services/keystore/ocr.go

```diff
@@ -134,20 +137,26 @@ func (ks *ocr) Export(id string, password string) ([]byte, error) {
 	return key.ToEncryptedJSON(password, ks.scryptParams)
 }
 
-func (ks *ocr) EnsureKey() (ocrkey.KeyV2, bool, error) {
+// EnsureKey verifies whether the OCR key has been seeded, if not, it creates it.
+func (ks *ocr) EnsureKey() error {
 	ks.lock.Lock()
 	defer ks.lock.Unlock()
 	if ks.isLocked() {
-		return ocrkey.KeyV2{}, false, ErrLocked
+		return ErrLocked
 	}
+
 	if len(ks.keyRing.OCR) > 0 {
-		return ocrkey.KeyV2{}, true, nil
+		return ErrOCRKeySeeded
```

Comment:
Same.
Just returning nil here is simpler.
And same comment for all other places too.